src/shared/constants.h

```diff
@@ -122,8 +122,8 @@ static const long ARDUINO_BAUD_RATE = 115200;
  */
 static const int ESTOP_MESSAGE_SIZE_BYTES = 1;
 
-static const unsigned char ESTOP_PLAY_MSG = 1;
-static const unsigned char ESTOP_STOP_MSG = 0;
+static const unsigned char ESTOP_PLAY_MSG = 0;
```

Comment:
yeah it is, i inverted the logic on the estop and missed this spot
the tests passed because they redefined their own ESTOP_STOP/PLAY constants

I removed those and pointed the test to these